modules/ROOT/pages/online-meetup.adoc

```diff
@@ -83,18 +83,81 @@ We also host regular online meetups! Check out the schedule below:
 |===
 |Date | Presenters | Description | Sign up
 
-|**19 October** +
-_7am PDT, 10am EDT, 2pm UTC, 4pm CEST, 7:30pm IST, 10pm AWST_
+|**5 November** +
```

Comment:
@adam-cowley I think we should be able to pull the data via the YT API automatically or?

Reply:
Yeah, we'd need an API key though.  Maybe something we can add to the build process to retrieve the latest videos?